File: Data Structures & Algorithms/best-time-to-buy-and-sell-stock-ii/submission-0.java
class Solution {
    public int maxProfit(int[] prices) {
        int profit=0;
        for(int day=0;day<prices.length-1;day++){
            if(prices[day] <prices[day+1]){
                profit+=Math.abs(prices[day]-prices[day+1]);
            }
        }
        return profit;

    }
}

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.py
class Solution:
    def maxProfit(self, prices: List[int]) -> int:
        profit=0 
        n=len(prices)
        for i in range(n):
            for j in range(i+1,n):
                num=prices[j]-prices[i]
                if num > profit : profit = num
        return profit 